lib/feeder/sync_episodes.ex

```diff
@@ -0,0 +1,33 @@
+defmodule Feeder.SyncEpisodes do
+
+  def sync(), do: sync(nil)
+  def sync(dtim) do
+    case get_episodes(dtim) do
+      {:error, err} -> {:error, err}
+      {_ok_or_partial, total, docs} ->
+        {created, updated} = update_episodes(docs)
+        {:ok, created, updated, total - created - updated}
```

Comment:
These return created/updated/remaining counts, to make it easy on the mix task doing the logging.